torch/_inductor/mkldnn_lowerings.py

```diff
@@ -544,24 +547,79 @@ def qlinear_unary(
             bias_dtype = None if bias is None else bias.get_dtype()
 
             choices: List[ChoiceCaller] = []
+
             if use_max_autotune():
+                # NOTE: since vector weight scale is supported with oneDNN qlinear in PyTorch,
+                # we are able to support the case of per-token quantized activation & per-channel quantized weights
+                # if x_zp is not present in a pattern, since in that case, we can apply the actual scale of x
```

Comment:
This line of comment seems relate to the pattern matcher but not here is the lowering phase. Why we need to mention it here?

Reply:
Because the pattern-replacement is done to accommodate oneDNN qlinear in the first place, and the GEMM template is also being forced to support only what oneDNN qlinear currently supports.

Reply:
Will add this info in the PR description instead. Thanks!